src/main/java/io/mosip/print/controller/Print.java

```diff
@@ -1,65 +1,92 @@
+	@PostMapping(value = "/enqueue", consumes = "application/json")
+	@PreAuthenticateContentAndVerifyIntent(secret = "Kslk30SNF2AChs2", callback = "/print/enqueue", topic = "http://mosip.io/print/pdf")
 	public void printPost(@RequestBody MOSIPMessage message) {
 		System.out.println(message.getTopic());
-		//TODO: Validate the MOSIPmessage
-		//TODO:Call the print service with the map that we received from MOSIPMessage
-		//printService.print()
+		// TODO: Validate the MOSIPmessage
+		// TODO:Call the print service with the map that we received from MOSIPMessage
+		// printService.print()
 	}
-    
-    @PostMapping(path = "/callback/notifyPrint", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-	//@ApiResponses(value = { @ApiResponse(code = 200, message = "Request authenticated successfully") })
-	@PreAuthenticateContentAndVerifyIntent(secret = "test", callback = "/print/callback/notifyPrint", topic  = "792112/CREDENTIAL_ISSUED")
-	public ResponseWrapper<?> handleSubscribeEvent( @RequestBody EventModel eventModel) {
-    	
-    	JSONArray jsonAray=new JSONArray(eventModel.getEvent().getData());
-    	JSONObject jsonObj=new JSONObject(jsonAray.get(0).toString());
-    	String credential=jsonObj.getString("credential");
-    	byte[] str1=Base64.getDecoder().decode(credential);
-    	System.out.println("Credential :  "+str1);
-		return new ResponseWrapper<>();
+
+	/**
+	 * Gets the file.
+	 *
+	 * @param printRequest the print request DTO
+	 * @param token        the token
+	 * @param errors       the errors
+	 * @param printRequest the print request DTO
+	 * @return the file
+	 * @throws IOException
+	 * @throws RegPrintAppException the reg print app exception
+	 */
+	@PostMapping(path = "/callback/notifyPrint", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+	@PreAuthenticateContentAndVerifyIntent(secret = "Kslk30SNF2AChs2", callback = "/v1/print/print/callback/notifyPrint", topic = "${mosip.event.topic}")
+	public ResponseEntity<Object> handleSubscribeEvent(@RequestBody EventModel eventModel) throws IOException {
+		String credential = eventModel.getEvent().getData().get("credential").toString();
+		byte[] str1 = CryptoUtil.decodeBase64(credential);
+		String decodedCrdential = new String(str1, Charset.forName("UTF-8"));
+		JSONObject jsonObject = new JSONObject(decodedCrdential);
+		Map proofMap = new HashMap<String, String>();
+		proofMap = (Map) eventModel.getEvent().getData().get("proof");
+		String sign = proofMap.get("signature").toString();
+		byte[] pdfbytes = printService.getDocuments(decodedCrdential, eventModel.getEvent().getTransactionId(),
+				getSignature(sign, credential), "UIN", false)
+				.get("uinPdf");
+		InputStreamResource resource = new InputStreamResource(new ByteArrayInputStream(pdfbytes));
+		/*
+		 * File pdfFile = new File(
+		 * "/media/lenovo/872da60f-3c16-4cfb-b900-0f63cbe7f3a9/opt/projects/mosip/MajorBug/print/uin.pdf"

```

Comment:
Comment with local file path should not be here.

Reply:
These are the file as-as from `mosip/print`. The only place where I made modification is [here](https://gist.github.com/typelogic/01d9971096837aa8b8165eeb15a5fd33)